assets/messages.yml

```diff
@@ -473,6 +473,22 @@ messages:
 
         %quick_links%
       fillname: []
+  attach-new-attachment:
+    - project:
+        - bds
+        - mc
+        - mcd
+        - mce
+        - mcl
+        - mcpe
+        - realms
+      name: Attach new attachment
+      shortcut: attachnew
+      message: |-
+        Your report contained an attachment with an invalid extension so it was removed. 
```

Comment:
This sounds like "your report was removed" :thinking:

Would it be possible to mention the file name of the attachment? Something like `Your attachment (%s%) was removed because it has an invalid extension`.

Reply:
Might be hard as we add a single comment for multiple attachments

Reply:
changed the message